### ISSUE TITLE
App Image not loading

### ISSUE THREAD

**Issue by Msouza91** [ytmdesktop/ytmdesktop#60]
I've opened the app image on Fedora 30, the page doesn't load and I'm unable to listen to music, page loads normally on browsers. The configuration screens load correctly.

**Comment by SN9NV** [comment]
Can confirm. Running from the command line, I get the error:
```
$ ./YouTube.Music.Desktop.App.1.5.0.AppImage
/tmp/.mount_YouTubuG72dY/youtube-music-desktop-app: error while loading shared libraries: libgobject-2.0.so.0: cannot open shared object file: No such file or directory
```

**Comment by adlerluiz** [comment]
@rickinbsb can you check this please?

**Comment by rickinbsb** [comment]
on it

**Comment by rickinbsb** [comment]
I've uploaded a new build, can you see if it works?

**Comment by SN9NV** [comment]
I'm downloading the AppImage from the Releases tab on Github. The issue persists with 1.5.2

```
./YouTube.Music.Desktop.App.1.5.2.AppImage
/tmp/.mount_YouTubH5TKaB/youtube-music-desktop-app: error while loading shared libraries: libgobject-2.0.so.0: cannot open shared object file: No such file or directory
```

**Comment by mingjun97** [comment]
Which desktop enviornment are you useing? i.e. gnome/xfce/kde etc.

**Comment by SN9NV** [comment]
It's stock Ubuntu 18.04.3, so Gnome 3.28.2

**Comment by SN9NV** [comment]
I have discovered what is wrong. Electron-Builder is only set up to build 32bit AppImage's for Linux. It doesn't look like I can install a 32-bit `libglib2`.

I can confirm that the AppImage built by copying the Linux config to `electron-builder64.yml` and running `yarn dist:x64` does produce a working AppImage. I'm not sure if the CI process is capable of making a 32 and 64-bit version of the AppImage.

Should I make a PR with my changes?

**Comment by mingjun97** [comment]
Sure. Thanks for your contribute!

**Comment by rickinbsb** [comment]
Thanks @SN9NV! Pull request merged. The fix and the working AppImage will go online on 1.5.3

**Comment by SN9NV** [comment]
Great thanks. You can mention me on future Linux issues if you want me to see what I can do. :+1:

**Comment by rickinbsb** [comment]
Fixed on 1.5.3
